sirepo/package_data/static/js/sirepo-components.js

```diff
@@ -3172,10 +3180,22 @@ SIREPO.app.directive('sbatchOptions', function(appState) {
                 appState.watchModelFields($scope, [$scope.simState.model + '.' + e], trimHoursAndCores);
             });
 
+            $scope.connectionStatusMessage = function () {
+                if  (authState.sbatchLoginSuccess === undefined) {
+                    return null;
+                }
+                return authState.jobRunModeMap[appState.models[$scope.simState.model].jobRunMode]
+                    + (authState.sbatchLoginSuccess ? '' : ' no') + ' connection established';
```

Comment:
This is where string files would let us define the message nicely. Mixing presentation and business logic yield wonky error messages. Rather say:

* Connected to v.radia.run
* Not connected to v.radia.run

Reply:
https://github.com/radiasoft/sirepo/pull/2523/commits/2c505a1eca579dafbef6921f946a8bc6a642cdc8